README.md

```diff
@@ -146,3 +146,49 @@ let response = await api.updateBug(123456, {
 ```
 
 Returned value is same as described in Bugzilla [docs](https://bugzilla.readthedocs.io/en/latest/api/core/v1/bug.html#update-bug).
+
+## Retrieving attachments by ID
+
+```javascript
+// .getAttachment() accepts one parameter, ID of attachment, as number
+let attachment = await api.getAttachment(123456);
+```
+
+Return value is Attachment object.
+
+## Retrieving all attachments of bug
+
+```javascript
+// .getBugsAttachments() accepts one parameter, ID of bug, as number
+let attachments = await api.getBugsAttachments(123456);
+```
+
+Return value is array of Attachment objects.
+
+## Creating attachments
+
+```javascript
+let attachment = await api.createAttachment(123456, {
+  ids: [123456, 123457];
+  data: "base64-content";
```

Comment:
Do you think it would be more useful to callers to allow passing a Buffer or an ArrayBuffer and then convert to base-64 internally?

Reply:
@Mossop I updated PR, please have a look - https://github.com/Mossop/bugzilla-ts/pull/81/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R284-R286